modules/monitoring-adding-a-secret-to-the-alertmanager-configuration.adoc

```diff
@@ -138,3 +138,49 @@ endif::openshift-dedicated,openshift-rosa[]
 
 . Save the file to apply the changes to the `ConfigMap` object. The new configuration is applied automatically.
 
+.Examples
+
+These examples show settings that configure Alertmanager in the `openshift-monitoring` namespace to authenticate with a webhook endpoint by using an API token that is configured in a `Secret` object.
+
+The following sample YAML shows a `Secret` object with authentication credentials to be referenced in the Alertmanager config map settings for the Cluster Monitoring Operator (CMO):
+
+[source,yaml]
+----
+apiVersion: v1
+data: 
+  api_token: <api-token> <1>
+kind: Secret
+metadata: 
+  name: alertmanager-credentials <2>
+  namespace: openshift-monitoring
+----
+<1> Replace with the API token used to authenticate to the webhook endpoint.
+<2> Reference this name in the `secrets` field in the CMO config map settings. 
+
+The following sample YAML shows CMO config map settings for Alertmanager. These settings reference the authentication credentials contained in the `Secret` object, which is mounted as a volume in the `alertmanager` container for the Alertmanager pods:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    alertmanagerMain:
+      enabled: true
+      secrets:
+      - alertmanager-credentials <1> 
+      receivers:
+        - name: webhook
+        webhook_configs: 
+        - url: 'https://webhook.example.com/' <2>
+          http_config: 
+            authorization: 
+              type: apikey
+              credentials_file: /etc/alertmanager/secrets/alertmanager-credentials/api_token # /etc/alertmanager/secrets/<secret name>/<key name> <3>
```

Comment:
would it be more explicit to use the example secret name + key name? e.g. etc/alertmanager/secrets/alertmanager-credentials/api_token # /etc/alertmanager/secrets/alertmanager-credentials/api_token?

Otherwise use `<secret name>` and `<key name>` in the secret manifest?